### DISCUSSION TITLE
Alerts and Hurt page time period changing

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#6454] [Unsupported Versions] [answered]
Hi!
After changing on of time-related fields on Alerts or Hunt page, search starts working and I am not able to change second field until search stopped. For example, I need to change search from 3 hours to 1 day. My steps:
- change 3 to 1;
- wait for search to stop;
- change hours to days;
- wait for correct result.

If time periods are long, search can last for a minute, what is a little annoying.
Maybe search should starts manually?
Thanks!

**Comment by jertel** [comment, marked as answer]
There is an option called "Auto Hunt" mentioned in the [docs](https://docs.securityonion.net/en/2.3/hunt.html?highlight=auto%20hunt#auto-hunt). It's enabled by default, but can be switched off.

**Reply by Sahale** [reply]
Wow, didn't see it. Thanks!

**Reply by Sahale** [reply]
Sorry, but Alerts page doesn't have this option, only Hunt.
